File: dayTwo/python/after_break/father_twice_sons_age.py
def is_double_sons_age(number_one, number_two):
    years_to_double = 0
    if(number_one/2 > number_two):
        years_to_double = number_one - (number_two * 2)
        print("The father was twice his son's age ", years_to_double, " years ago")
    
    elif(number_one/2 < number_two):
        years_to_double = (number_two * 2) - number_one
        print("The father will be twice his son's age in ", years_to_double," years time")
    


fathers_age = 38
sons_age = 18
is_double_sons_age(fathers_age, sons_age)

File: dayTwo/javascript/after_break/fatherTwiceSonsage.js

function isDoubleSonsAge(numberOne, numberTwo){
    let yearToDouble = 0;
    if(numberOne/2 > numberTwo){
        yearToDouble = numberOne - (numberTwo * 2);
        console.log("The father was twice his son's age "+yearToDouble+" years ago");
    }
    else if(numberOne/2 < numberTwo){
        yearToDouble = (numberTwo * 2) - numberOne;
        console.log("The father will be twice his son's age in "+yearToDouble+" years time");
    }
}
isDoubleSonsAge();
const fathersAge = 38;
const sonsAge = 18;
isDoubleSonsAge(fathersAge, sonsAge);
